### PR TITLE
Feature/search song on youtube

### PR DESCRIPTION
**Pull request by goody153**
## Ticket
 - [add search video](https://trello.com/c/hMCV6lEY)

## Summary:
 - Added template search.html
 - Added the search form on the header of base.html
 - Added new method on YoutubeApi.py
 - Added new class on views.py

### CODE REVIEW

File: playlist/views.py [hunk 1]
```diff
@@ -138,3 +139,22 @@ def get(self, *args, **kwargs):
...
+    """
+    template_name = 'playlist/search.html'
+
+    def post(self, *args, **kwargs):
+        # get the data from youtube api
+        searches = self.search_list_by_keyword(
+            self.authenticate_yt(),
+            part='snippet',
+            maxResults=25,
+            q=self.request.POST.get('youtube_keyboard'),
```
Comment (mcandidier) [inline playlist/views.py]: assign `self.request.POST.get('youtube_keyboard')` to variable.

Comment (mcandidier) [inline playlist/views.py]: what is this? youtube_keyboard?

File: assets/js/search.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+//doesn't allow to search if textfield is empty
+  $(document).on('mouseover', '#search_form', function(){
+    if($('#search_field').val() !== '')
```
Comment (mcandidier) [inline assets/js/search.js]: this is wrong,  if else statement should expected to have {} to determined code blocks

 e.g: if (foo){
} else {
 pass
}

Comment (goody153) [reply assets/js/search.js]: Updated

**Pull request merged**